appendix/teachmeqiskit2018/README.md

```diff
@@ -0,0 +1,18 @@
+## Teach Me QISKit Award 2018: Top Submissions
+
+This folder contains the top submissions of the Teach Me QISKit Award 2018.
+Most of the tutorials were written and tested with qiskit-sdk-py version 0.4.15.
```

Comment:
qiskit-core as the repo name is going and the tag is still in qiskit-core

Reply:
Done